client/web/src/enterprise/batches/create/RunServerSideModal.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+
+import { VisuallyHidden } from '@reach/visually-hidden'
+import CloseIcon from 'mdi-react/CloseIcon'
+
+import { Button, Link, Modal } from '@sourcegraph/wildcard'
+
+import styles from './RunServerSideModal.module.scss'
+
+export interface RunServerSideModalProps {
+    name: string
+    originalInput: string
+    isLightTheme: boolean
+    setIsRunServerSideModalOpen: (condition: boolean) => void
+}
+
+export const RunServerSideModal: React.FunctionComponent<RunServerSideModalProps> = ({
+    name,
+    originalInput,
+    isLightTheme,
+    setIsRunServerSideModalOpen,
+}) => (
+    <Modal
+        onDismiss={() => {
+            setIsRunServerSideModalOpen(false)
+        }}
+        aria-labelledby={MODAL_LABEL_ID}
+        className={styles.modal}
+    >
+        <div>
+            <h3 id={MODAL_LABEL_ID}>Running batch changes server side is not enabled</h3>
```

Comment:
I see this in the original, too, but it seems like we're hyphenating "server side" most of the time?

```suggestion
            <h3 id={MODAL_LABEL_ID}>Running batch changes server-side is not enabled</h3>
```